src/cdk/activation.ts

```diff
@@ -68,4 +70,14 @@ async function registerCdkCommands(context: vscode.ExtensionContext, explorer: A
             }
         })
     )
+
+    const visualizationManager = new AslVisualizationCDKManager(context)
+    context.subscriptions.push(
+        vscode.commands.registerCommand('aws.cdk.renderStateMachineGraph', async (node: ConstructNode) => {
+            return await visualizationManager.visualizeStateMachine(
```

Comment:
We should actually be calling the renderGraph command here, and have that retrieve the visualization from the visualization manager.